input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,395 @@
+- **Partial compliance**: An AU Core compliant resource meets all mandatory requirements but fails to meet some optional constraints of the compared IPA, IPS, or US Core profile. Flagged with an <img src="orange_checkmark.svg.png" width="20"/>.
+- **Non-compliance**: An AU Core compliant resource fails to meet one or more mandatory requirements of the compared IPA, IPS, or US Core profile. Flagged with a <img src="cross_red_circle.svg.png" width="20"/>.
+
+Where partial compliance or non-compliance is declared, more information is provided in the sections below. Blank cells in the table indicate that the given AU Core profile has no equivalent profile.
+
+<table border="1" style="width: 100%; margin: auto; border-collapse: collapse;">
+    <thead>
+        <tr>
+            <th style="width: 25%; text-align: center; vertical-align: middle;"><a href="https://build.fhir.org/ig/hl7au/au-fhir-core">AU Core</a></th>
+            <th style="width: 25%; text-align: center; vertical-align: middle;"><a href="https://hl7.org/fhir/uv/ipa/STU1/index.html">International Patient Access 1.0.0</a></th>
+            <th style="width: 25%; text-align: center; vertical-align: middle;"><a href="https://hl7.org/fhir/uv/ips/STU1.1/index.html">International Patient Summary 1.1.0</a></th>
+            <th style="width: 25%; text-align: center; vertical-align: middle;"><a href="https://hl7.org/fhir/us/core/STU7/">US Core 7.0.0</a></th>
+        </tr>
+    </thead>
     <tbody>
         <tr>
-            <td width="93" valign="top">
-                <p>
-                    <strong>
-                        <a href="https://build.fhir.org/ig/hl7au/au-fhir-core">
-                            AU Core
-                        </a>
-                    </strong>
-                </p>
-            </td>
-            <td width="191" colspan="2" valign="top">
-                <p>
-                    <a href="https://hl7.org/fhir/uv/ipa/STU1/">
-                        <strong>International Patient Access 1.0.0</strong>
-                    </a>
-                    <strong></strong>
-                </p>
-            </td>
-            <td width="198" colspan="2" valign="top">
-                <p>
-                    <a href="https://hl7.org/fhir/uv/ips/STU1.1/">
-                        <strong>International Patient Summary 1.1.0</strong>
-                    </a>
-                    <strong></strong>
-                </p>
-            </td>
-            <td width="215" colspan="2" valign="top">
-                <p>
-                    <a href="https://hl7.org/fhir/us/core/STU6.1/">
-                        <strong>US Core 6.1.0</strong>
-                    </a>
-                    <strong></strong>
-                </p>
-            </td>
+            <td style="width: 25%; text-align: center; vertical-align: middle;"><a href="StructureDefinition-au-core-allergyintolerance.html">AU Core AllergyIntolerance</a></td>
```

Comment:
In terms of the flag & svg used, you will need to go through anything that was red cross to check if it's truly incompatible or it's a yellow tick.

Reply:
Done.